app/helpers/automate_tree_helper.rb

```diff
@@ -1,30 +1,51 @@
 module AutomateTreeHelper
   def at_tree_select_toggle(edit_key)
-    build_ae_tree(:automate, :automate_tree)
+    edit_key == :method ? build_ae_tree(:ae_methods, :automate_tree) : build_ae_tree(:automate, :automate_tree)
```

Comment:
imho this method is a bit too long, how about breaking it down to multiple smaller methods?

Reply:
I made it a little smaller, but we are planning to replace this old dialog with a new component and this helper will disappear. So I think that is not necessary to refactor it at this time. (https://www.pivotaltracker.com/n/projects/1613907/stories/151605408)